### ISSUE TITLE
Left-hand sidebar to not scroll down with page

### ISSUE THREAD

**Issue by abhigenie92** [AmmsA/theresumator#27]
Would it be possible to make the left-hand sidebar to stay in place while the content on right scroll downs.

**Comment by abhigenie92** [comment]
Any ideas if this feature would make sense?

**Comment by AmmsA** [comment]
Yeah.. this is more like a feature than an issue.. If you can code it nicely, feel free to make a pull request, but please deploy it and send me a link so I can see it live.

Cheers.
